source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/ImageRenderer.java

```diff
@@ -139,16 +139,25 @@ else if ((image = Image.dynamic_cast(baseCardElement)) == null)
         imageView.setTag(image);
 
         String imageBackgroundColor = image.GetBackgroundColor();
-        int backgroundColor = -1;
+        int backgroundColor = 0;
         if(!TextUtils.isEmpty(imageBackgroundColor))
         {
-            try
+            // check that it has 9 characters and that the color string isn't a color name
+            if(imageBackgroundColor.length() == 9 && imageBackgroundColor.charAt(0) == '#')
             {
-                backgroundColor = Color.parseColor(imageBackgroundColor);
+                try
+                {
+                    // if the color string is not valid, parseColor throws a IllegalArgumentException
+                    backgroundColor = Color.parseColor(imageBackgroundColor);
+                }
+                catch (IllegalArgumentException e)
+                {
+                    backgroundColor = 0;
+                }
```

Comment:
Should we issue a warning? Should this be done at the objectModel level so that we can have parse warnings?